chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1184,6 +1184,10 @@ public synchronized void loadChunks(TaskTracker progress, World world,
       waterOctree.endFinalization();
     }
 
+    for (Entity entity : actors) {
```

Comment:
For the sake of completeness, this should probably also be called for entities that are not actors. Although I don't know any entities that would need block information?

Reply:
I'll go ahead and add another loop for it, just to future-proof it.